lms/static/js/bookmarks/views/bookmark_button.js

```diff
@@ -1,6 +1,6 @@
 ;(function (define, undefined) {
     'use strict';
-    define(['gettext', 'jquery', 'underscore', 'backbone', 'js/views/message_banner'],
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'lmsCoursewareMessageBanner'],
         function (gettext, $, _, Backbone, MessageBannerView) {
```

Comment:
In fact, if you switch back to using paths then there will be no need to change the Jasmine configuration.